src/aotcompile.cpp

```diff
@@ -412,7 +411,6 @@ void *jl_create_native_impl(jl_array_t *methods, LLVMOrcThreadSafeModuleRef llvm
         // finally, make sure all referenced methods also get compiled or fixed up
         jl_compile_workqueue(params, policy);
```

Comment:
I think it'd be interesting to parallelize the jl_emit_codeinst call some lines above this; iirc that was occupying a good amount of time during precompilation after the LLVM portion was parallelized. Might also serve to quickly root out any threading bugs.

Reply:
That call doesn't take any significant time. My plan was to move this code around later, so that it can more properly call addModule earlier, when each subgraph is complete, as LLVM is where the time is mostly spent.

Reply:
Maybe I'm misremembering and it was the entirety of this for loop that was expensive... still I think this for loop should be parallel since addModule doesn't get called here as part of image generation.